VideoLocker/res/layout/fragment_course_unit_only_on_youtube.xml

```diff
@@ -0,0 +1,45 @@
+<LinearLayout xmlns:android="http://schemas.android.com/apk/res/android"
+    xmlns:app="http://schemas.android.com/apk/res-auto"
+    xmlns:tools="http://schemas.android.com/tools"
+    android:layout_width="match_parent"
+    android:layout_height="match_parent"
+    android:background="@color/white"
```

Comment:
Good catch. Looks like this needs to be modified in the other layout as well. All the more reason to have a common layout.